pyproject.toml

```diff
@@ -24,6 +23,12 @@ dependencies = [
     "urllib3>=1.26,<3",
     "pyecma376-2>=0.2.4"
 ]
+# This line indicates that the version is dynamically determined
+dynamic = ["version"]
+
+[tool.setuptools_scm]
+# This section configures setuptools_scm
```

Comment:
This comment is obvious as well. 
Please add comments to what's the idea behind the configuration and what it achieves.